Mac/MainWindow/Timeline/TimelineViewController.swift

```diff
@@ -940,7 +940,7 @@ extension TimelineViewController: NSTableViewDelegate {
 				return [action]
 
 			@unknown default:
-				os_log(.error, "Unknown table row edge: %ld", edge.rawValue)
+				print("Unknown table row edge: \(edge.rawValue)")
```

Comment:
Is there a reason this is a print instead of os_log? Could this be os_log too?

Reply:
It seems most viewControllers use `print` instead of `os_log` for logging. I'm wondering if there's any reason to opt out `os_log` for UI-related codes and should I replace all those `print`s to `os_log`s or keep them as-is?

Reply:
Good point. Since we already have the precedent of using print for view controllers, let’s keep using print. Which means this change is the right thing to do, and I no longer am requesting changes. :)